buildpack/nginx.py

```diff
@@ -290,6 +290,7 @@ def get_access_restriction_locations():
                 # Escape special characters
                 issuer = issuer.replace(" ", "\\040")
                 issuer = issuer.replace(".", "\\.")
+                issuer = issuer.replace("'", "\\'")
```

Comment:
This will actually become `\'` instead of `'` - is this the intention?

Reply:
I beleave it should render `\'` in the nginx.conf and the check should work for `'`

Reply:
Okay. Maybe it's good to create an integration test which verifies this. It's quite easy to spin up an application, apply a path restriction and do a HTTP request with an Issuer DN header set.